product_order_noname/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2020 Vauxoo, S.A. de C.V.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    'name': 'Product Order No Name',
+    'summary': 'Speedup product retrieve',
+    "version": "12.0.1.0.0",
+    "development_status": "Production/Stable",
+    "category": "Technical Settings",
+    "website": "https://github.com/OCA/product-attribute",
+    "author": "Vauxoo, Odoo Community Association (OCA)",
+    "maintainers": ["WR-96", "moylop260"],
```

Comment:
Could you add my username also, please? :)

Reply:
No

What did you do?

hahaha

Reply:
I did a Review :).

But it doesn't matter if the maintainer didn't do anything, the important part is that they understand well the module and they are willing to review/contribute.

It's part of a campaign named "adopt an addon", see https://github.com/OCA/oca-github-bot/issues/93